.sdkmanrc

```diff
@@ -0,0 +1 @@
+java=11.0.14.1-ms
```

Comment:
Is this Microsoft's flavour of java? Should `Corretto-11.0.15.9.1` be used instead to match CI and PROD?

Reply:
@akash1810 unfortunately I don't think AWS publish Corretto 11 for M1 (only 17+). I'll double check this though.

Reply:
Update: https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html shows an M1-friendly version! For some reason, sdkman does not support it. I might look into raising a quick PR on sdkman to see if this can be supported.

Reply:
`asdf` supports it 😉